aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -304,6 +333,7 @@ spec aptos_framework::staking_contract {
         let accumulated_rewards = total_active_stake - staking_contract.principal;
         aborts_if !exists<stake::StakePool>(pool_address);
         aborts_if active + pending_active > MAX_U64;
+        // TODO: These function causes the timeout
```

Comment:
Thanks for marking this out! In this case, it is the non-linear arithmetic operations that causes long verification time.

Reply:
yeah, without assuming it should timeout, by the way, assume it in the range of 0-100 seemed not right, so it better stay as todo for now.